src/main/java/roomescape/reservation/domain/Reservation.java

```diff
@@ -2,31 +2,65 @@
 
 import java.time.LocalDate;
 import java.util.Objects;
-import roomescape.time.domain.ReservationTime;
+import roomescape.reservationTime.domain.ReservationTime;
+import roomescape.theme.domain.Theme;
 
 public class Reservation {
 
     private final Long id;
     private final String name;
     private final LocalDate date;
     private final ReservationTime time;
+    private final Theme theme;
 
-    public Reservation(final Long id, final String name, final LocalDate date, final ReservationTime time) {
+    private Reservation(final Long id,
+                        final String name,
+                        final LocalDate date,
+                        final ReservationTime time,
+                        final Theme theme
+    ) {
         this.id = id;
-        this.name = validateNonBlank(name);
-        this.date = date;
-        this.time = time;
+        this.name = Objects.requireNonNull(name, "이름은 null 일 수 없습니다.");
+        validateNameIsBlank(name);
+        this.date = Objects.requireNonNull(date, "날짜는 null 일 수 없습니다.");
+        this.time = Objects.requireNonNull(time, "예약 시간은 null 일 수 없습니다.");
+        this.theme = Objects.requireNonNull(theme, "테마는 null 일 수 없습니다.");
     }
 
-    private String validateNonBlank(final String name) {
-        if (name == null || name.isBlank()) {
-            throw new IllegalArgumentException("[ERROR] 이름은 빈 값일 수 없습니다.");
+    private static void validateNameIsBlank(final String name) {
```

Comment:
```suggestion
    private void validateNameIsBlank(final String name) {
```
static이 아니여도 동작할 것 같아요~

Reply:
맞네요! 수정하였습니다.